vnext/Desktop.DLL/react-native-win32.x86.def

```diff
@@ -46,6 +46,7 @@ EXPORTS
 ?typeName@dynamic@folly@@QBEPBDXZ
 ?updateProperties@ShadowNode@react@facebook@@UAEX$$QBUdynamic@folly@@@Z
 ?utf16ToUtf8@unicode@react@facebook@@YG?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@ABV?$basic_string@_WU?$char_traits@_W@std@@V?$allocator@_W@2@@5@@Z
+?makeMemoryMappedBuffer@JSI@Microsoft@@YG?AV?$shared_ptr@VBuffer@jsi@facebook@@@std@@QB_WI@Z
```

Comment:
>?ma [](start = 0, length = 3)

(nit) alphabetize

Reply:
It looks like we need to do so for all the def files. I'll send out a separate PR for this.

---
In reply to: [320920138](https://github.com/microsoft/react-native-windows/pull/3059#discussion_r320920138) [](ancestors = 320920138)